test/test_framework.py

```diff
@@ -1527,3 +1528,90 @@ def test_named_indicated_all(self, fake_stderr):
     def test_named_indicated_hook(self, fake_stderr):
         # The framework indicated the special value 'hook', nothing to do with any named call.
         self.check_trace_set('hook', 'mybreak', 0)
+
+
+@patch('sys.stderr', new_callable=io.StringIO)
```

Comment:
given the recommendation to minimize the surface area being mocked, wouldn't it make sense to patch sys.stderr at the same time as pdb.runcall rather than all the way here?

Reply:
good idea, done!